valve-firmware/valve-firmware.spec

```diff
@@ -0,0 +1,47 @@
+Name: valve-firmware
```

Comment:
This naming is consistent with other Fedora firmware packages: `${VENDOR}-firmware`.

```
$ sudo dnf search firmware
```